### ISSUE TITLE
Task [fonts] will not work with last bootstrap-sass 3.3.5

### ISSUE THREAD

**Issue by viniciusdacal** [yeoman/generator-webapp#338]
they remove fonts from bower.main
https://github.com/twbs/bootstrap-sass/commit/e62a79b1a7bfa4b12059638a47501b670deffa67#diff-0a08a7565aba4405282251491979bb6b

**Comment by silvenon** [comment]
Thanks for reporting :+1: I will see what's the best way to deal with this…

**Comment by jaromero** [comment]
A (temporary) workaround is now provided by Bootstrap, using `overrides` in bower.json:

http://blog.getbootstrap.com/2015/06/15/bootstrap-3-3-5-released/#wiredep-and-bower

From my tests, it works fine with the current wiredep.

Whenever bower decides to provide [a `files` field as well](https://github.com/bower/bower.json-spec/issues/47) and bootstrap makes use of it, these overrides can be removed.

**Comment by jaromero** [comment]
(Apologies for the multiple notifications)

I've updated the _bower.json template [here](https://github.com/jaromero/generator-gulp-webapp/commit/bdc2a80eb541b39406d340faf472a2e42fedd700). I'm not convinced by the tests I added though, since I couldn't find a way to actually test for the result of the `fonts` task, only whether the bower.json file had the proper overrides (and even then, maybe I should've just checked for "overrides" instead of every single added line).

If the overrides workaround is acceptable though, and if this looks good, I'll just go ahead and submit the PR.
